cilium-dbg/cmd/service_update.go

```diff
@@ -205,10 +204,6 @@ func updateService(cmd *cobra.Command, args []string) {
 		if resp.Status == nil {
 			Fatalf("Unable to retrieve cilium configuration: empty response")
 		}
-
-		if resp.Status.DatapathMode != datapathOption.DatapathModeLBOnly {
```

Comment:
Aren't backend weights still not supported for all datapath modes other than `veth` in combination with  `loadbalancer-only`? Or has something changed in the meantime and this is also supported for other datapath modes and we can really remove it completely?

```
option.Config.DatapathMode != datapathOption.DatapathModeVeth || !option.Config.LoadBalancerOnly
```

and maybe adjusting the error message?

Reply:
This is independent of veth or netkit. The weights support currently only has an exposure through the cilium-dbg cli command line, so you need to configure it manually, if you don't configure it specifically then each backend has equal weight. So removing this conditional here is fine.